pkg/api/vlabs/types.go

```diff
@@ -226,32 +254,6 @@ type KeyVaultCertificate struct {
 	CertificateStore string `json:"certificateStore,omitempty"`
 }
 
-// OrchestratorType defines orchestrators supported by ACS
-type OrchestratorType string
```

Comment:
OrchestratorType has a list of allowed orchestrators for sure. We just don't need a custom type for it. It will be checked in validation.
True, It should be OrchestratorVersionHint, thx.